shopfloor/services/cluster_picking.py

```diff
@@ -474,6 +480,14 @@ def scan_line(self, picking_batch_id, move_line_id, barcode):
             move_line, message=self.msg_store.barcode_not_found()
         )
 
+    def _get_prefill_qty(self, move_line, qty=False):
+        """Returns the increment qty or qty_done depending on the no_prefill_qty
```

Comment:
pls follow this convention for multiline docstrings https://peps.python.org/pep-0257/#multi-line-docstrings

Reply:
done